### DISCUSSION TITLE
Search extended to additional fields

### DISCUSSION THREAD

**Discussion by pandalamp** [mateusz-bak/openreads#772] [General]
I am interested in what other Openreads users think about extending search queries to cover additional fields besides author and title.

The advantage to how the search works currently is that it is very quick and predictable with directly relevant results. Extending it to include any or all arbitrary fields could reduce its utility, especially when tags and filters can already do much of what extended search could.

That said, tags don't make sense for finding books from a specific _Publication Year_. I use the _Notes_ field for marking which country a book was from, which would be convenient for me to search. People might want to search a character name listed in a _Description_ or _My Review_. But I concede these could all be edge cases against a feature already working correctly for most users.

Does anyone care to share their thoughts?

**Comment by lucastucious** [comment]
Hi there. I was currently unable to search for all books that have a specific tag. 

As I understand that you don't want to search for tag by default, I think having a custom field search could be awesome. Like discord did it 

Like `tag:searchedTag` or `note:"Some words to search"`
